Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.cpp

```diff
@@ -97,11 +97,6 @@ void JSWebAssemblyArray::finishCreation(VM& vm)
 {
     Base::finishCreation(vm);
     ASSERT(inherits(info()));
-
```

Comment:
but we didn't change the JSWebAssemblyArray supertype, right? So isn't this wrong?

Reply:
This is about the `seal` right? The issue is that this doesn't work without structure transitions, which the wasm structure in this patch doesn't support. The `seal` is also unnecessary with the JS API patch (https://github.com/WebKit/WebKit/pull/10408), so it was removed here assuming that other patch would also land.

That said, if we apply @Constellation's suggestion to put the RTT in the object instance, it's not necessary to remove this line in this patch.